### ISSUE TITLE
SessionScope fails on Electron

### ISSUE THREAD

**Issue by mihaiblaga89** [supertokens/supertokens-website#98]
```
let sessionScope = normaliseSessionScopeOrThrowError(getWindowOrThrow().location.hostname);
if (options.sessionScope !== undefined) {
    sessionScope = normaliseSessionScopeOrThrowError(options.sessionScope);
}
```
On Electron `getWindowOrThrow().location.hostname` returns an empty string and that fails the check. Even if I have set the `sessionScope` in config the location takes precedence. Would be nice for the config to be the one taken into account first.

```
const sessionScope = normaliseSessionScopeOrThrowError(options?.sessionScope || getWindowOrThrow().location.hostname);
```

If you're ok with it I can make a PR.

**Comment by rishabhpoddar** [comment]
Thanks for pointing this out! I'll make the change and release a new version for this sometime tomorrow.

**Comment by rishabhpoddar** [comment]
Fix released in v10.0.6
